db/migrate/20230407170430_add_payload_credentials_configuration_scripts.rb

```diff
@@ -0,0 +1,11 @@
+class AddPayloadCredentialsConfigurationScripts < ActiveRecord::Migration[6.1]
+  def change
+    add_column    :configuration_scripts, :payload,       :jsonb
+    add_column    :configuration_scripts, :credentials,   :jsonb
+    add_column    :configuration_scripts, :context,       :jsonb
+    add_column    :configuration_scripts, :output,        :jsonb
+    add_column    :configuration_scripts, :status,        :string
+    add_reference :configuration_scripts, :miq_task,      :type => :bigint, :index => true
+    add_column    :configuration_scripts, :run_by_userid, :string
```

Comment:
Renamed this to `run_by_userid` from just `userid` so it is more obvious what this column is